react/components/ProductImages/components/Carousel/index.js

```diff
@@ -209,7 +224,7 @@ class Carousel extends Component {
       resistanceRatio: slides.length > 1 ? 0.85 : 0,
       ...(showNavigationArrows && {
         renderNextButton: () => (
-          <span className={`swiper-caret-next pl7 pr2 right-0 ${caretClassName}`}>
+          <span className={`swiper-caret-next pl7 pr2 right-0 ${caretClassName} ${cssHandles.swiperCaret}`}>
```

Comment:
Should also have a different handle to differentiate right from left

example:
`${cssHandles.swiperCaret} ${cssHandles.swiperCaretRight}`